Descent3/lnxmain.cpp

```diff
@@ -337,105 +325,6 @@ int SDLCALL d3SDLEventFilter(void *userdata, SDL_Event *event) {
 
 void StartDedicatedServer();
 
-#ifdef __PERMIT_MAKEHOG
```

Comment:
sidenote: we should get rid of all those __PERMIT defines, none of them are bound to CMake definitions anymore

Reply:
Agreed.